contrib/flo-styx/src/main/java/com/spotify/flo/contrib/styx/StructuredLogMessage.java

```diff
@@ -40,13 +41,13 @@
   @JsonProperty @Nullable String styx_execution_id();
   @JsonProperty @Nullable String styx_trigger_id();
   @JsonProperty @Nullable String styx_trigger_type();
-  @JsonProperty @Nullable String flo_task_id();
```

Comment:
Why removing the `flo_` prefix here?

Reply:
to make it easier later when filtering luigi task logs
so we don’t need to differentiate
which could be tricky for UI work.

Reply:
Right. But `framework` and `task_id` feel a bit generic. Wondering if we should at least qualify them with `workflow_` or maybe nest them in a `"workflow": { ... }` object.

```yaml
"workflow": {
  "framework": "flo",
  "task_id": "my-task#54354",
  ...
}
```

Reply:
maybe just prefix with `workflow_`? no strong need to go nested i think.

Reply:
those `styx_` things also fit in to `workflow` object if we want to go nested way.

Reply:
https://github.com/spotify/flo/pull/93